ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/users/UsersController.java

```diff
@@ -691,59 +693,50 @@ private Account modifyAccount(Account account, ServletInputStream inputStream) t
 		JSONObject json;
 		try {
-			json = new JSONObject(strUser);
+			json = new JSONObject(FileUtils.asString(is));
 		} catch (JSONException e) {
 			LOG.error(e.getMessage());
 			throw new IOException(e);
 		}
 
-		String givenName = RequestUtil.getFieldValue(json, UserSchema.GIVEN_NAME_KEY);
-		if(givenName == null){
-			throw new IllegalArgumentException(UserSchema.GIVEN_NAME_KEY + " is required" );
-		}
-		String surname= RequestUtil.getFieldValue(json, UserSchema.SURNAME_KEY);
-		if(surname == null){
-			throw new IllegalArgumentException(UserSchema.SURNAME_KEY + " is required" );
-		}
-		String email= RequestUtil.getFieldValue(json, UserSchema.MAIL_KEY);
-		if(email == null){
-			throw new IllegalArgumentException(UserSchema.MAIL_KEY + " is required" );
-		}
-
-		String postalAddress =  RequestUtil.getFieldValue(json, UserSchema.POSTAL_ADDRESS_KEY );
-
-		String postOfficeBox =  RequestUtil.getFieldValue(json, UserSchema.POST_OFFICE_BOX_KEY );
-
-		String postalCode = RequestUtil.getFieldValue(json, UserSchema.POSTAL_CODE_KEY);
-
-		String street= RequestUtil.getFieldValue(json, UserSchema.STREET_KEY);
-		String locality = RequestUtil.getFieldValue(json, UserSchema.LOCALITY_KEY);
-
-		String phone = RequestUtil.getFieldValue(json, UserSchema.TELEPHONE_KEY);
-
-		String facsimile = RequestUtil.getFieldValue( json, UserSchema.FACSIMILE_KEY);
-
-		String title = RequestUtil.getFieldValue( json, UserSchema.TITLE_KEY);
-
-		String description = RequestUtil.getFieldValue( json, UserSchema.DESCRIPTION_KEY);
-
-		String manager = RequestUtil.getFieldValue(json, UserSchema.MANAGER_KEY);
-
-		String uid;
-		try {
-			uid = createUid(givenName, surname);
-		} catch (DataServiceException e) {
-			LOG.error(e.getMessage());
-			throw new IOException(e);
-		}
+		String givenName     = RequestUtil.getFieldValue(json, UserSchema.GIVEN_NAME_KEY);
+		String surname       = RequestUtil.getFieldValue(json, UserSchema.SURNAME_KEY);
+		String email         = RequestUtil.getFieldValue(json, UserSchema.MAIL_KEY);
+		String postalAddress = RequestUtil.getFieldValue(json, UserSchema.POSTAL_ADDRESS_KEY );
+		String postOfficeBox = RequestUtil.getFieldValue(json, UserSchema.POST_OFFICE_BOX_KEY );
+		String postalCode    = RequestUtil.getFieldValue(json, UserSchema.POSTAL_CODE_KEY);
+		String street        = RequestUtil.getFieldValue(json, UserSchema.STREET_KEY);
+		String locality      = RequestUtil.getFieldValue(json, UserSchema.LOCALITY_KEY);
+		String phone         = RequestUtil.getFieldValue(json, UserSchema.TELEPHONE_KEY);
+		String facsimile     = RequestUtil.getFieldValue(json, UserSchema.FACSIMILE_KEY);
+		String title         = RequestUtil.getFieldValue(json, UserSchema.TITLE_KEY);
+		String description   = RequestUtil.getFieldValue(json, UserSchema.DESCRIPTION_KEY);
+		String manager       = RequestUtil.getFieldValue(json, UserSchema.MANAGER_KEY);
+		String org           = RequestUtil.getFieldValue(json, UserSchema.ORG_KEY);
+
+		if(givenName == null)
```

Comment:
Validation is only used for end user input, admin users (MOD_LDAPADMIN) should be able to edit user without restriction.